### ISSUE TITLE
Settings page chrome://settings/privacy doesn't show up

### ISSUE THREAD

**Issue by jstkdng** [Eloston/ungoogled-chromium#930]
**Describe the bug**
Settings page chrome://settings/privacy doesn't exist on ungoogled chromium 80.0.3987.87, it does on regular chromium.

**To Reproduce**
1. Go to "Settings"
2. Click on "Privacy and security"

**Expected behavior**
Privacy settings shows up

**Environment (please complete the following information):**
 - OS/Platform and version: Arch Linux Rolling
 - ungoogled-chromium version: 80.0.3987.87

**Comment by jstkdng** [comment]
I get this messages on the console when opening the settings page:
```
[79114:79114:0207/071810.224685:ERROR:CONSOLE(222)] "Unexpected condition on chrome://settings/: Could not find value for enableSafeBrowsingSubresourceFilter", source: chrome://resources/js/load_time_data.js (222)
[79114:79114:0207/071810.224765:ERROR:CONSOLE(222)] "Unexpected condition on chrome://settings/: [undefined] (enableSafeBrowsingSubresourceFilter) is not a boolean", source: chrome://resources/js/load_time_data.js (222)
[79114:79114:0207/071810.291253:ERROR:CONSOLE(11)] "Uncaught Error: Assertion failed: Pref is missing: safebrowsing.scout_reporting_enabled", source: chrome://resources/polymer/v1_0/polymer/polymer-extracted.js (11)
[79114:79114:0207/071826.080454:ERROR:CONSOLE(166)] "Uncaught TypeError: Cannot read property 'offsetTop' of undefined", source: chrome://settings/settings_main/settings_main.js (166)
```

**Comment by tangalbert919** [comment]
I'll take a look into that when I have the time. I definitely missed something as I was getting these patches updated.

**Comment by kramred** [comment]
On macOS with ungoogled chromium 80.0.3987.87 the "Privacy and security" settings __do__ show up, have you tried a fresh/empty profile folder to make sure it's not related to your settings?

**Comment by jstkdng** [comment]
@kramred 
yes, I tested on a new profile since I had to reinstall my system.

**Comment by PF4Public** [comment]
Same on Gentoo. "Privacy" is missing and first settings group is "Language" instead.

**Comment by kramred** [comment]
Actually, I did not pay proper attention and when I pasted the link `chrome://settings/privacy` something showed up but it's __not the "Privacy and security" settings__ (it's the just the normal Settings `chrome://settings` page). So, it's also __not__ working on macOS.

**Comment by Eloston** [comment]
Should be fixed with 80.0.3987.87-2.
